text/0010-exception-types.md

```diff
@@ -0,0 +1,75 @@
+* Start Date: 2022-09-17
+* RFC Type: feature
+* RFC PR: 0010
+* RFC Status: draft
+
+# Summary
+
+This RFC suggests a feature which introduces additional types of exceptions next to `unhandled`.
+
+# Motivation
+
+Currently, exception which cause the running software to exit are marked as `unhandled: true`. This isn't enough for SDKs where an exception can be unhandled but at the same time doesn't cause the software to exit.
+
+Sentry shows exceptions which aren't caught but also do not cause the software to exit in the same way as exceptions which are manually caught by a developer. This seems rather unintuitive and makes exceptions seem less severe than they are.
+
+This issue impacts for example the [Dart/Flutter SDK](https://github.com/getsentry/sentry-dart/issues/456), the Unity SDK, the React Native SDK and possibly more.
+
+Another issue is, that excpetions which don't cause the application to exit but are uncaught, are not considered in the [`session health` metric](https://develop.sentry.dev/sdk/sessions/).
+
+# Proposal
```

Comment:
I'd add 2 more items to the proposal:

A way to filter events on the issues page or discover, such as:

`handled:yes`, `handled:no` are already possible.
Introducing a new one which filters the `process termination` type, eg `handled:termination` or a brand new tag, that's just an example.

The other 2 is the UI label highlight for `unhandled` events and `process termination`.
Currently, there's an `unhandled` label on the issue's page but it's only highlighted for `process termination` errors.

Reply:
Added ✅